README.md

```diff
@@ -11,46 +11,60 @@ ember install ember-feature-flags
 
 ### Usage
 
-This addon injects a property `features` (configurable) into your routes, controllers and components.
+This addon provides a service named `features` (configurable) available for injection into your routes, controller,s components, etc.
 
 For example you may check if a feature is enabled:
 
 ```js
-export default Ember.Controller.extend({
-  plans: function(){
-    if (this.features.isEnabled('new-billing-plans')){
+export default Controller.extend({
+  features: service(),
+
+  plans: computed('features.new-billing-plans', function(){
```

Comment:
This would need to be `computed('features.newBillingPlans')` ... the feature flags are camelized when turned into bound properties.

Reply:
Perhaps then there should be another example of `isEnabled` use, though...